tests/test_checkout.py

```diff
@@ -218,3 +224,37 @@ def test_checkout_create_order_insufficient_stock(
     checkout = Checkout(request_cart, customer_user, 'tracking_code')
     with pytest.raises(InsufficientStock):
         checkout.create_order()
+
+
+@pytest.mark.parametrize('view', [
+    summary_without_shipping,
+    summary_with_shipping_view,
+    anonymous_summary_without_shipping,
+])
+@patch('saleor.checkout.views.summary.get_address_form',
+       return_value=(Mock(), Mock()))
+@patch('saleor.checkout.views.summary.BillingWithoutShippingAddressForm')
+@patch('saleor.checkout.views.summary.get_billing_forms_with_shipping',
+       return_value=(Mock(), Mock(), None))
+def test_note_ins_summary_views(
+        mock_get_billing_forms_with_shipping,
+        mock_BillingWithoutShippingAddressForm, mock_get_address_form, view):
+    checkout = Checkout(Mock(), Mock(
+        default_billing_address=Mock(),
+        addresses=Mock(is_authenticated=Mock(return_value=True))),
+        'tracking_code')
+    rf = RequestFactory()
+    request = rf.post('/test/', {'note': 'test_note'})
+    request.user = Mock()
+    request.country = Mock()
+    mock_BillingWithoutShippingAddressForm().is_valid.return_value = False
```

Comment:
Can we use `lower_case_with_underscores` naming style here?

Reply:
I'd also consider:

* always passing a `spec` to `Mock` instances, otherwise they have all possible parameters and we may be masking a bug in the tested code
* using the `monkeypatch` fixture instead of `mock.patch` in cases where it results in less code